### PR TITLE
added the biochem user comment collecting/posting service

### PR DESCRIPTION
**Pull request by qzzhang**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -375,7 +375,7 @@ app.get('/v0/list/*', AuthRequired, function(req, res) {
 
     var mailOptions = {
         from: 'help@modelseed.org',
-        to: 'help@modelseed.org',         // list of receivers
+        to: 'qzhang@anl.gov',         // list of receivers
```
Comment (samseaver) [inline server.js:379]: Are you actively using this for your testing? I want to be careful that we don't forget to revert the email address when deploying, so can you revert this change? Thanks.

Comment (qzzhang) [reply server.js:379]: Yes.  That's my noted point.  For now, I don't want to bombard people on the list of help@modelseed.org before we decide to deploy it into prod.

Comment (samseaver) [reply server.js:379]: I understand that, but my point is that you don't need to include this in the PR, you can change it locally.
